bundles/org.openhab.binding.homewizard/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -4,27 +4,60 @@
 	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
 	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
 
-
 	<thing-type id="p1_wifi_meter">
 		<label>HomeWizard Wi-Fi P1 Meter</label>
 		<description>This thing provides the measurement data that is available through the http interface of the HomeWizard
 			Wi-Fi P1 Meter.</description>
 
 		<channels>
-			<channel id="total_energy_import_t1" typeId="total_energy_import_t1"/>
-			<channel id="total_energy_import_t2" typeId="total_energy_import_t2"/>
-			<channel id="total_energy_export_t1" typeId="total_energy_export_t1"/>
-			<channel id="total_energy_export_t2" typeId="total_energy_export_t2"/>
+			<channel id="active_current" typeId="system.electric-current">
+				<label>Current</label>
+				<description>The sum of the current for all phases</description>
+			</channel>
+			<channel id="active_current_l1" typeId="system.electric-current">
+				<label>Current L1</label>
+			</channel>
+			<channel id="active_current_l2" typeId="system.electric-current">
+				<label>Current L2</label>
+			</channel>
+			<channel id="active_current_l3" typeId="system.electric-current">
+				<label>Current L3</label>
+			</channel>
 			<channel id="active_power" typeId="active_power"/>
 			<channel id="active_power_l1" typeId="active_power_l1"/>
 			<channel id="active_power_l2" typeId="active_power_l2"/>
 			<channel id="active_power_l3" typeId="active_power_l3"/>
+			<channel id="active_voltage" typeId="system.electric-voltage">
+				<label>Active Voltage</label>
+			</channel>
+			<channel id="active_voltage_l1" typeId="system.electric-voltage">
+				<label>Active Voltage L1</label>
+			</channel>
+			<channel id="active_voltage_l2" typeId="system.electric-voltage">
+				<label>Active Voltage L2</label>
+			</channel>
+			<channel id="active_voltage_l3" typeId="system.electric-voltage">
+				<label>Active Voltage L3</label>
+			</channel>
+
+			<channel id="any_power_failures" typeId="power_failures"/>
```

Comment:
The word "any" gives me the impression that it's a boolean value, but actually it's the count of power failures (long or short), if I understand it correctly. Perhaps the name could be rephrased to better express this? Perhaps `power_failure_count` and `long_power_failure_count`, respectively? Or simply `power_failures` and `long_power_failures` (which is already consistent with channel below)?

Reply:
yes, thanks, that is much better. When building up a model from the API, one tend to re-use those conventions.